gn2/wqflask/templates/show_trait_details.html

```diff
@@ -222,6 +222,7 @@
         <button type="button" class="btn btn-default" title="Check probe locations at UCSC" onclick="window.open('{{ UCSC_BLAT_URL }}', '_blank')">Verify</button>
         {% endif %}
         {% if this_trait.symbol != None %}
+	<button type="button" class="btn btn-default" title="Write or review comments about this gene" onclick="window.open('http://localhost:5012/genewiki/{{ this_trait.symbol }}', '_blank')">(GN2) GeneWiki</button>
```

Comment:
I don't this  is okay . What will happen if the port we are currently using changes? I would  recommend you  use  a dynamic path obtained from the server.

Question: Is it okay   to expose the port that a service is running  to the client?

Reply:
Thanks for catching this :D. You're right, this is not ok.

Reply:
Done. Comment addressed.